index.html

```diff
@@ -16,15 +16,17 @@
   <body>
     <!-- 헤더 -->
     <header class="header">
-      <a href="/public" class="logoContainer">
+      <div class="headerInner">
+      <a href="./" class="logoContainer">
         <img
           class="logoImg"
           src="src/shared/img/logo/brand/logo.svg"
-          alt="logo"
+          alt="logoImg"
```

Comment:
💊 제안  
img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때  
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 가능한 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주는 것이 좋습니다.

```suggestion
          alt="판다마켓"
```